### DISCUSSION TITLE
Debug adapter ?

### DISCUSSION THREAD

**Discussion by gautaz** [OmniSharp/omnisharp-vscode#7129] [Q&A] [answered]
Hello,

Looking at the list of [debug adapters implementations](https://microsoft.github.io/debug-adapter-protocol/implementors/adapters/), `vscode-csharp` is referenced as `adapter` and `usage` for C# (`adapter` points in fact to https://github.com/OmniSharp/omnisharp-vscode which redirects to https://github.com/dotnet/vscode-csharp).

Is there any documentation in the repository or somewhere else detailing how anyone can connect a DAP client to this implementation of a debug adapter for C#?

**Comment by gregg-miskelly** [comment, marked as answer]
The debugger that ships as part of vscode-csharp is licensed to only work in Visual Studio Code / Visual Studio. If you are looking for an open source debug adapter for C#, see https://github.com/Samsung/netcoredbg.

**Reply by gautaz** [reply]
Oh thank you @gregg-miskelly, would you mind if I ask the people maintaining the [DAP project](https://github.com/microsoft/debug-adapter-protocol) to add `netcoredbg` as a debug adapter option for C#?

**Reply by gregg-miskelly** [reply]
It is already listed on that table. It is named "Debugger for .NET Core".

**Reply by gautaz** [reply]
You're right, I missed it, thanks for pointing it out.
